service/history/execution/mutable_state_task_generator.go

```diff
@@ -791,7 +792,7 @@ func (r *mutableStateTaskGeneratorImpl) GenerateFromCrossClusterTask(
 	var targetCluster string
 
 	sourceDomainEntry := r.mutableState.GetDomainEntry()
-	if !sourceDomainEntry.IsDomainActive() && !sourceDomainEntry.IsDomainPendingActive() {
+	if isActive, _ := domain.IsActive(sourceDomainEntry, r.clusterMetadata); !isActive && !sourceDomainEntry.IsDomainPendingActive() {
```

Comment:
I guess you are skipping err check here because you need implementation details?

Reply:
We do not need error here. Both `GetNotActiveErr` and `IsActive` are now combined into one function. The error part is not relevant here.